README.rst

```diff
@@ -66,6 +67,7 @@ Then:
 * Activate your project on `pyup.io`_.
 
 .. _`pip docs for requirements files`: https://pip.pypa.io/en/stable/user_guide/#requirements-files
+.. _Register: https://packaging.python.org/distributing/#register-your-project
```

Comment:
This link seems to not be pointing to a section about registering project anymore.
Maybe we should just tell people to [use twine](https://github.com/pypa/twine) to register the project?

Reply:
That link nows goes to a location that specifies using twine to register the project.